### PR TITLE
[prerelease] dash 1.19.0

### PR DESCRIPTION
**Pull request by harryturr**
changelog | loosen versions | dash v1.19.0 | renderer v1.9.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +20,28 @@ to close the error messages box.
   - updated doc string for `assets_external_path` Dash argument to be more clear that it will allways be joined with the `assets_url_path` argument when determining the url to an external asset.
 - [#1493](https://github.com/plotly/dash/pull/1493) Fix [#1143](https://github.com/plotly/dash/issues/1143), a bug where having a file with one of several common names (test.py, code.py, org.py, etc) that imports a dash component package would make `import dash` fail with a cryptic error message asking whether you have a file named "dash.py"
 
+## DASH CORE COMPONENTS
```
Comment (rpkyle) [inline CHANGELOG.md:23]: Do we normally capitalize these? I think it's probably OK just to go with Dash Core Components :slightly_smiling_face:

Comment (rpkyle) [inline CHANGELOG.md:23]:
```suggestion
## Dash Core Components
```

File: dash-renderer/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "dash-renderer",
-  "version": "1.8.3",
+  "version": "1.9.0",
```
Comment (rpkyle) [inline dash-renderer/package.json:3]: Did we already note the renderer update in the CHANGELOG?

**Review by rpkyle — APPROVED** [review]
A couple minor comments, but otherwise looks :ok_hand: 

:woman_dancing:

File: CHANGELOG.md (hunk 1 above)
Comment (harryturr) [reply CHANGELOG.md]: https://github.com/plotly/dash/pull/1535/commits/6de7f9c749a648079f19fb3a2c331357953c4a43

Comment (harryturr) [reply CHANGELOG.md]: https://github.com/plotly/dash/pull/1535/commits/6de7f9c749a648079f19fb3a2c331357953c4a43

File: dash-renderer/package.json (hunk 2 above)
Comment (harryturr) [reply dash-renderer/package.json:3]: Updated "Dash" to "Dash and Renderer" in https://github.com/plotly/dash/pull/1535/commits/6de7f9c749a648079f19fb3a2c331357953c4a43

**Pull request merged**